File: parcial02/punto02/java/batch/src/main/java/co/edu/eafit/st0257/s20181/Line.java
package co.edu.eafit.st0257.s20181;

public class Line {
   private static int nro = 0;
   private int nLine;
   public Line(Card card) {
      nLine = Line.nro++;
   }
   public String toString() {
      return "Line: " + nLine;
   }
}

File: parcial02/punto02/c++/src/line.cpp
#include "line.h"

int Line::nro = 0;

Line::Line(Card& aCard) :
  nLine(Line::nro++) { }

std::ostream &operator<<(std::ostream& strm, const Line line) {
  strm << "Line: " << line.nLine << std::endl;
}

